services/state/addressbook/node.proto

```diff
@@ -137,4 +137,12 @@ message Node {
      * node SHALL fail.
      */
     bool deleted = 9;
+
+    /**
+    * An administrative key for the node address book entry.
+    * <p>
+    * This key MUST sign each transaction to update this node.<br/>
+    * This field SHALL be set and SHALL NOT be an empty `KeyList`.
```

Comment:
Here also, it should be a valid key, since we should check in code

Reply:
Updated. Thanks @Neeharika-Sompalli